### ISSUE TITLE
Checkout Fails When YITH Subscriptions Premium is active for membership products

### ISSUE THREAD

**Issue by andrewlimaza** [strangerstudios/pmpro-woocommerce#98]
When checking out with PayPal for a subscription product and PMPro WooCommerce product that assigns a membership level gets the following error at checkout - "Invalid Regular period. You must specify valid values for the A3, P3, and T3 parameters for a subscription."

**Comment by eighty20results** [comment]
Since YITH Subscriptions and WooCommerce Subscriptions are two different subscription solutions (and AFIK we only handle WooCommerce Subscriptions based actions in the integration add-on), I would expect us to have to add YITH Subscriptions support before we can even begin to troubleshoot this issue.

**Comment by ideadude** [comment]
You are correct @eighty20results. I asked @andrewlimaza to open this issue for us to explore what would be needed to support YITH. It might be as simple as hooking YITH hooks into the callbacks we already have (or slightly altered ones). It might be harder.
